src/routes/controllers/articles.controller.js

```diff
@@ -52,3 +52,85 @@ export default async function create(req, res) {
     return errorResponse(res, 401, error.message);
   }
 }
+
+/**
+ * Fetch all articles
+ * @param {Object} req Express Request Object
+ * @param {Object} res Express Response Object
+ * @returns {Object} res with articles array if it exists
+ * @returns {Object} res with 404 response if the array is empty
+ */
+export async function getAll(req, res) {
```

Comment:
let's remove this controller as it's not related to this ticket.

Reply:
Okay. I will implement this now